storage/pebble/registers.go

```diff
@@ -61,7 +92,17 @@ func (s *Registers) Store(
 	height uint64,
 	entries flow.RegisterEntries,
 ) error {
-	//  TODO: add value check once LatestHeight is implemented
+	s.lock.Lock()
+	defer s.lock.Unlock()
+	if height == s.latestHeight {
+		// already updated
+		return nil
```

Comment:
This if statement won't allow the storage to index height 0 when initialized empty, since the value of height will be set at 0 and then possibly we want to index block at height 0.